cmd/cmd.go

```diff
@@ -759,14 +792,82 @@ func generateInteractive(cmd *cobra.Command, model string, wordWrap bool, format
 			fmt.Printf("Unknown command '%s'. Type /? for help\n", args[0])
 		}
 
+		line, images, err := extractFileNames(line)
+		if err != nil {
+			return err
+		}
+
+		if len(images) > 0 {
+			opts.Images = images
+			ctx := cmd.Context()
+			ctx = context.WithValue(ctx, generateContextKey("context"), []int{})
```

Comment:
this looks like a bug maybe? why does images being set cause history to be cleared?